src/execution/DAG/dag.c

```diff
@@ -651,10 +637,10 @@ int RedisAI_DagRun_Reply(RedisModuleCtx *ctx, RedisModuleString **argv, int argc
         persist_status = _DAG_PersistTensors(ctx, rinfo);
     } else {
         if (rinfo->dagOps[0]->commandType == REDISAI_DAG_CMD_MODELRUN) {
-            persist_status = _ModelSingleOp_PersistTensors(ctx, rinfo->dagOps[0], rinfo->err);
+            persist_status = _PersistTensors(ctx, rinfo->dagOps[0], rinfo->err);
```

Comment:
This if-else is redundant now as we use the same call for both model and script cases, no?

Reply:
yes, good catch